express-hello-world/index.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+
+const app = express();
+
+app.use(function (req, res, next) {
+  // console.log('Time:', Date.now())
+  // console.log(req.method)
+  // console.log(res.method)
+  res.send('test send');
+  next();
+});
+
+app.listen(3000, () => {
+  // console.log("listening on port 3000")
```

Comment:
If an exercise requires you to leave in a console log you can put the following comment above it so it will pass the commit checks:

```JAVASCRIPT
// eslint-disable-next-line no-console
console.log("listening on port 3000");
```